package com.tbless.inventoryManagementApp.utils;

public class ExceptionUtils {
    public static final String USER_REGISTRATION_FAILED = "Registration failed for user with username %s";
    public static final String USER_ALREADY_EXISTS = "%s already exists";
    public static final String USER_NOT_FOUND= "User not found";
    public static final String USER_WITH_ID_NOT_FOUND = "User with id %d not found";
    public static final String USER_WITH_USERNAME_NOT_FOUND = "User with username %s not found";
    public static final String USER_STILL_HAS_AN_UNSETTLED_LOAN = "You still have an unsettled loan";
    public static final String FAILED_TO_ADD_CARD = "You cannot add the same card twice";
    public static final String INVALID_CARD = "Card number is invalid";
    public static final String NO_CARD_FOUND = "No Card found";
    public static final String PRODUCT_DOES_NOT_EXIST = "Product with id %d does not exist";
    public static final String THIS_PRODUCT_DOES_NOT_EXIST = "Product %s does not exist";
    public static final String YOU_DO_NOT_HAVE_ANY_PRODUCT_AT_THE_MOMENT = "You do not have any product at the moment";
    public static final String PRODUCT_ALREADY_EXISTS = "Product with name %s already exist";
    public static final String USER_WITH_USERNAME_DOES_NOT_EXIST = "User with username %s does not exist";

    public static final String PROFILE_UPDATE_FAILED="Profile Update Failed";
}

